framework/CrossValidations/Factory.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on Sept 5 2017
+
+@author: wangc
+"""
+#for future compatibility with Python 3-----------------------------------------
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+if not 'xrange' in dir(__builtins__):
+  xrange = range
+#End compatibility block for Python 3-------------------------------------------
+
+################################################################################
+from utils import utils
+from .CrossValidation import CrossValidation
+from .SklCrossValidation import SciKitLearn
+## [ Add new class here ]
+################################################################################
+## Alternatively, to fully automate this file:
+# from OutStreamManagers import *
```

Comment:
we can delete this (or better link the description to this class)

Reply:
Ok, I will delete this.